libs/cardano-ledger-core/testlib/Test/Cardano/Ledger/Core/JSON.hs

```diff
@@ -68,3 +70,14 @@ goldenJsonPParamsSpec =
   it "Golden JSON specs for PParams " $ \file -> do
     decoded <- eitherDecodeFileStrict @(PParams era) file
     void $ expectRightExpr decoded
+
+goldenJsonPParamsUpdateSpec ::
+  forall era.
+  EraTest era =>
+  SpecWith FilePath
+goldenJsonPParamsUpdateSpec =
+  it "Golden JSON specs for PParamsUpdate" $ \file -> do
+    let ppu = runGen 100 100 (arbitrary @(PParamsUpdate era))
```

Comment:
This is a bit fragile, since that means the golden test will depend on `Arbitrary` instance of `PParamsUpdate`, which can change even through QuickCheck version update. However, this is fine for now. It is better than nothing, which is what we have today :grin:

Reply:
I added an issue to create `FromJSON` instances, add `PParamsUpdate` to roundtrip tests and make the golden tests more robust: https://github.com/IntersectMBO/cardano-ledger/issues/5052